src/.clang-format

```diff
@@ -1,9 +1,10 @@
 Language:        Cpp
 AccessModifierOffset: -4
-AlignAfterOpenBracket: false
+AlignAfterOpenBracket: true
```

Comment:
Slightly -0 on this:

* Wastes whitespace of N*M, where N is the length of the function name and M the number of parameters in their own line.
* Changing the function name forces a re-flow of *all* parameters and/or arguments (if they are in multiple lines). Thus increasing the size of patches.

Also, I am wondering if this change is needed, given that you also set `AllowAllArgumentsOnNextLine` to true?

Reply:
> Wastes whitespace

I don't understand what 'wastes whitespace' means. Either the code is easier to read or not.

Without this setting, then clang would format arguments on subsequent lines as follows:

```
size_t CScheduler::getQueueInfo(std::chrono::system_clock::time_point& first,
    std::chrono::system_clock::time_point& last) const
```

which is _definitely_ less clear, since the arguments are now aligned with the function body and there's no obvious visual split between the function signature and body.

> Changing the function name forces a re-flow of all parameters and/or arguments

I think maybe you misunderstand this setting. It only affects that are _already_ on a newline. It doesn't move parameters onto their own line. So:

- function renames are already rare
- most functions don't have parameters split between lines so are unaffected
- for those that do, run clang-format and review ignoring whitespace.

Reply:
Fair enough